### PR TITLE
Update currency-converter.md

### PR DESCRIPTION
**Pull request by ahmagdy**
Extend the project description with more concrete points

### CODE REVIEW

File: projects/currency-converter.md [hunk 1]
```diff
@@ -22,62 +22,55 @@ It is expected that your app will have at least these features and functionality
...
 ---
 
 ### `JS3 week3` - Classes, Promises advanced
 
-- [ ] rest, revise JS fundamentals, finish, and brush up your project, prepare to present and explain it.
-- [ ] make improvements to the style or functionality or add additional functionality.
-
+- [ ] Rest, revise JS fundamentals, finish, and brush up your project, prepare to present and explain it.
+- [ ] Make improvements to the style or functionality or add additional functionality.
+- [ ] Testing
```
Comment (shpomp) [inline projects/currency-converter.md]: Testing could move to the previous week?

File: projects/currency-converter.md [hunk 2]
```diff
@@ -22,62 +22,55 @@ It is expected that your app will have at least these features and functionality
 
 ### `JS2 week1` - Browser environment, DOM manipulation, DOM event listeners
 
-- create a form to insert a new currency rate.
+- Create a form to insert a new currency rate.
 
-- implement one of the two (or both!)
+- Implement one of the two (or both!)
```
Comment (shpomp) [inline projects/currency-converter.md]:
```suggestion
// TODO: describe the task to implement currency conversion. 
- Implement one of the two (or both!)
```
At this point we have not asked to implement the currency conversion yet, so I suppose "Update existing currency conversion with a new rate" falls out of context if I understand it correctly.

**Review by shpomp — APPROVED** [review]
Fantastic! Thank you very much! ❤️ 
Left a few notes.

(The capitalization, hehe 😅 )

**Comment by ahmagdy** [comment]
> Fantastic! Thank you very much! ❤️ Left a few notes.
> 
> (The capitalization, hehe 😅 )

Happy to help :D

File: projects/currency-converter.md (hunk 1 above)
Comment (ahmagdy) [reply projects/currency-converter.md]: I'm flexible. I moved it to the previous week and added more context

File: projects/currency-converter.md (hunk 2 above)
Comment (ahmagdy) [reply projects/currency-converter.md]: Good point. Added an explicit ask about the conversion

**Pull request merged**